### PR TITLE
Updating getting started doc

### PR DESCRIPTION
**Pull request by secondsun**
## Motivation

https://github.com/aerogear/aerogear-android-sdk/issues/46

JIRA: https://issues.jboss.org/browse/AGDROID-757

## Description
I've added some sections and clarifications to the Getting Started doc.

## Progress

- [x] Done Task

## Additional Notes

@wei-lee @david-martin  I've tagged you both as reviewers since I saw issues from both of you on this issue.

### CODE REVIEW

File: docs/getting-started.adoc [hunk 1]
```diff
@@ -3,9 +3,94 @@
...
+=== MavenLocal()
+
+If you are using a build of the SDK from source you will need to add mavenLocal() to the build.gradle file in your project's root directory.
+
+```groovy
+allprojects {
+    repositories {
+        google()
+        jcenter()
+        mavenLocal() // <-- Add This line
```
Comment (wei-lee) [inline docs/getting-started.adoc]: to help with testing the lib after we publish a new version, should this be added as the first repo?

**Comment by TommyJ1994** [comment]
@secondsun Small thing. Links are in markdown format for an .adoc file, so they don't render in GH.

File: docs/getting-started.adoc (hunk 1 above)
Comment (secondsun) [reply docs/getting-started.adoc]: I'm not sure why it would help with testing, I think I might be missing something.  Can you elaborate?

**Comment by secondsun** [comment]
@TommyJ1994 I've fixed the Markdown/asciidoc drift.  Can you take a look?

File: docs/getting-started.adoc [hunk 2]
```diff
@@ -3,9 +3,101 @@
...
+cd aerogear-android-sdk
+./gradlew install
+----
+
+You can verify the install went well by checking your local maven repository
+
+----
+ls ~/.m2/repository/org/aerogear
+
+#Should output auth  core  metrics
```
Comment (danielpassos) [inline docs/getting-started.adoc]: metrics is not a module anymore, so the output should be only core

File: docs/getting-started.adoc [hunk 3]
```diff
@@ -3,9 +3,101 @@
...
+In the build.gradle file for your app you can begin adding dependencies provided by the sdk.
+
+----
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation 'com.android.support:appcompat-v7:27.0.2'
+    implementation 'com.android.support.constraint:constraint-layout:1.0.2'
+    implementation 'org.aerogear:core:5.0.0-SNAPSHOT'
+    implementation 'org.aerogear:auth:5.0.0-SNAPSHOT' //If you are using IDM services provided by KeyCloak
+    implementation 'org.aerogear:metrics:5.0.0-SNAPSHOT' //If you are using our metrics services
```
Comment (danielpassos) [inline docs/getting-started.adoc]: There is not metrics module anymore

File: docs/getting-started.adoc [hunk 4]
```diff
@@ -48,10 +140,14 @@ subprojects {
...
 }
 ----
 
 == Importing individual SDK
 
 Please follow individual SDK's documentation to see how to add them to the project
+
+ * link:../core/README.md[Core]
+ * link:../auth/README.md[Auth] 
+ * link:../metrics/README.md[Metrics]
```
Comment (danielpassos) [inline docs/getting-started.adoc]: There is not metrics module anymore

Comment (secondsun) [reply docs/getting-started.adoc]: @danielpassos removed.  I hope.

**Pull request merged**